projects/arlas-components/src/lib/pipes/format-legend/format-legend.pipe.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Pipe, PipeTransform } from '@angular/core';
+export interface  LegendParamsResult {translKey: string; field?: string; normalized?: string; metric?: string; format: ParamsResultType;}
```

Comment:
Could you format it with one key per line please? :pray: 
I have two questions about the interface:
- why use `translKey` and not `translateKey` ?
- why did you choose to put normalized as a string and not as a boolean ? When a legend is normalised, it will always be the same thing we will display ie normalised or nothing; wouldn't it be simpler that way ?

Reply:
the idea was to return interpolate params for the translate pipe. That is why i chose a string over a boolean. Yes you re right !